website/docs/reference/datasources/querying-google-sheets.md

```diff
@@ -1,583 +1,272 @@
-
-### Query
-
-You use the Query tab to define - the type of operation you want to perform and the entity on which the operation should be performed.
-
-:::info
-Some operations aren't available for some entities. If an operation isn't available for an entity, the entity is grayed out, and the tooltip `Action not supported` is displayed when you hover over it.
-:::
-
-## **Operation**
-
-Operation lets you define the type of action you want to perform on Google Sheets (Spreadsheets or individual Sheets available in a spreadsheet). Following is the list of available operations:
-
-| **Operation**                                                             | **Description**                                                                           | **Available on below Entity** |
-| ------------------------------------------------------------------------- | ----------------------------------------------------------------------------------------- | ----------------------------- |
-| [**Fetch Details**](./querying-google-sheets.md#fetch-details)      | Use it to fetch details like the name of the spreadsheet, created date and time, and more | Spreadsheet                   |
-| [**Insert One**](./querying-google-sheets.md#insert-one)            | Use it to insert a single row in a sheet or add a new spreadsheet                         | Spreadsheet/Sheet Rows        |
-| [**Update One**](./querying-google-sheets.md#update-one)            | Use it to update rows in a given sheet in a spreadsheet                                   | Sheet Rows                    |
-| [**Delete One**](./querying-google-sheets.md#delete-one)            | Use it to delete a spreadsheet or a sheet or row in a sheet                               | Spreadsheet/Sheet/ Sheet Rows |
-| [**Fetch Many**](./querying-google-sheets.md#fetch-many-sheet-rows) | Use it to fetch data from a sheet or all the spreadsheets available in your account.      | Spreadsheet/Sheet Rows        |
-| [**Insert Many**](./querying-google-sheets.md#insert-many)          | Use it to insert multiple rows in a given sheet                                           | Sheet Rows                    |
-| [**Update Many**](./querying-google-sheets.md#update-many)        | Use it to update multiple rows in a given sheet                                           | Sheet Rows                    |
-
-## Fetch details
-
-The operation allows you to fetch the details of a given spreadsheet.
-
-<VideoEmbed host="youtube" videoId="elAabPXK_xk" title="Fetch Details of a Spreadsheet" caption="Fetch Details of a Spreadsheet"/>
-
-You can set the parameter values as below:
+<figure>
+  <img src="/img/google-sheets-query-page.png" style={{width: "100%", height: "auto"}} alt="Configuring a query from the query screen."/>
```

Comment:
Maybe this image should show the dropdown with all the queries?

Reply:
that section was removed, so the image was moved somewhere that makes more sense for what is shown